extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -366,9 +378,27 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       // holdback branch sends non-numeric value.
       return false;
     }
-    // eslint-disable-next-line no-undef
-    const defaultRange = IDLE_VIEWPORT_RANGE_3 ? 3 : 12;
-    return vpRange || defaultRange;
+
+    if (vpRange) {
+      return vpRange;
```

Comment:
Is it ok if we are using `vpRange` or `this.performance_ = null` but we are still reporting exp id / control id in ads metrics?